doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -399,6 +399,16 @@ The following strings must not be used as attribute names:
 
 * Virtual attribute names (see specific section on "Virtual Attributes")
 
+## Metadata names restrictions
+
+The following strings must not be used as metadata names:
+
+* `ngsi:onArrival`, as it has an special use (see "Notification metadata marks" section).
```

Comment:
This part has been re-done so probably the comment is obsolete. NTC, review the new version.